SharePic/app/src/main/java/edu/sdsu/cs/sharepic/activity/MainActivity.java

```diff
@@ -46,23 +49,38 @@ protected void onCreate(Bundle savedInstanceState) {
         listView.setAdapter(adapter);
 
         dropbox = Dropbox.getInstance(getApplicationContext());
```

Comment:
can we use "this" instead of getApplicationContext()? (Its also one of my queries)